app/controllers/cards_controller.rb

```diff
@@ -0,0 +1,48 @@
+class CardsController < ApplicationController
+  before_action :set_card, only: [:destroy, :edit, :update]
+  respond_to :html
+
+  def index
+    @cards = current_user.cards.all.order('review_date')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.